dpypelines/pipeline/shared/utility.py

```diff
@@ -0,0 +1,15 @@
+import os
+
+def enrich_online(message_creating_function):
+    def wrapper(*args):
```

Comment:
on this and the following line, can you pass through **kwargs as well please, of we're not already its a matter of time before we have a message function that uses a keyword or two.